src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithStrongNames.txt

```diff
@@ -211,7 +210,7 @@ Referenced assemblies:
 - 'Microsoft.VisualStudio.Interop, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.Shell.15.0, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
 - 'Microsoft.VisualStudio.Shell.Framework, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
-- 'Microsoft.VisualStudio.Threading, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
+- 'Microsoft.VisualStudio.Threading, Version=17.3.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
```

Comment:
As discussed offline, this worries me a bit - I'm not sure the new VSIX will work with VS2022RTM. I'll try it on the test VM.

Reply:
:-(

```
Still unable to load MEF component DLL: Could not load file or assembly 'Microsoft.VisualStudio.Threading, Version=17.3.0.0, ...
```

I'll trying switching to v17.0 of the SDK.

Reply:
I should've thought of that :(

Reply:
Update: the `MS.VS.Threading` assembly appears to be the only assembly that actually changed version number between SDK 17.0 and 17.3.
I installed an SLVS VSIX built with SDK 17.0 on the VS2022 RTM Azure VM and it worked correctly.